src/legacy/ui/public/vis/editors/default/components/sidebar/controls.tsx

```diff
@@ -82,45 +79,19 @@ function DefaultEditorControls({
           </EuiFlexItem>
 
           <EuiFlexItem grow={false}>
-            {isInvalid && isTouched ? (
-              <EuiToolTip
```

Comment:
@sulemanof Sorry, I gave bad advice, I forgot that the invalid version had a tooltip. This tooltip needs to stay in here to guide users to the fact that there's an error.

Reply:
OK, returned